src/postgres/src/test/regress/yb_pg_misc_serial_schedule

```diff
@@ -28,4 +24,3 @@ test: yb_pg_create_aggregate
 
```

Comment:
Please remove `yb_pg_create_aggregate` from here.  Also check whether the above dependencies are still needed now that this is gone.

Reply:
The dependencies will still be needed for the other tests.